.github/workflows/prune.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Prune
+
+on:
+  schedule:
+    - cron: "0 5 * * 3"
+  workflow_dispatch: {}
+
+jobs:
+  ghcr:
+    name: GitHub Container Registry
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Prune smoke test images
+        uses: snok/container-retention-policy@v2
+        with:
+          image-names: smoke-*
+          cut-off: One week ago UTC
+          keep-at-least: 1
+          account-type: org
+          org-name: ${{ github.repository_owner }}
+          token: ${{ secrets.GH_TOKEN }}
```

Comment:
hmm not sure we want this- it would run across all containers for our org? Feels like maybe not the right repo?

Reply:
Not sure I understand what you mean. Can you clarify?

Reply:
You are right, it should have different options.

Reply:
I think this should do it [`e78f1ef` (#38)](https://github.com/seamapi/fake-seam-connect/pull/38/commits/e78f1ef212a95fcde9c0aa2828ef364657767da6)